sopel/modules/meetbot.py

```diff
@@ -293,7 +296,7 @@ def meetingaction(bot, trigger):
     logplain('ACTION: ' + trigger.group(2), trigger.sender)
     logHTML_listitem('<span style="font-weight: bold">Action: </span>' + trigger.group(2), trigger.sender)
     meeting_actions[trigger.sender].append(trigger.group(2))
-    bot.say('ACTION: ' + trigger.group(2))
+    bot.say(bold('ACTION: ') + trigger.group(2))
```

Comment:
Switching to shorthand, since this is explained in depth above:

Space in bold